tests/test_vpc.py

```diff
@@ -228,7 +227,7 @@ def test_flow_logs_misconfiguration(self):
                                 "type": "flow-logs",
                                 "enabled": True,
                                 "op": "equal",
-                                "set-op": "or",
+                                "set-op": "and",
```

Comment:
this implies the behavior for set-op 'or' was actually 'and', we probably need to preserve for compatbility.

Reply:
ah it was always doing `and` on attributes, but the set-operator was whether the all of the flow logs matched for a resource versus any.